src/core/cli/man/content/streaming.md

```
@@ -0,0 +1,96 @@
+async def chat_endpoint(body: ChatRequest, chat: McpMeshTool = None) -> mesh.Stream[str]:
+    async for chunk in chat.stream(prompt=body.prompt):
+        yield chunk
+```
+
+Browsers consume via `fetch` + `ReadableStream` (`EventSource` is GET-only and won't work for POST endpoints). Plain JSON `mesh.route` handlers on the same agent are unaffected — only routes that declare a `Stream[str]` return type get the SSE wrapper.
+
+## Wire format
+
+The protocol is unmodified MCP. Any vanilla MCP client (Cursor, Claude Desktop, Cline, `fastmcp.Client`) can subscribe to chunks by passing a `progressToken` in `_meta` and registering a `progress_handler`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)